### PR TITLE
chore - Upgrade reth dependencies

### PR DESCRIPTION
**Pull request by debjit-bw**
upgraded to reth commit [9a732f6b7813f9ecb5f28b379d3062c287a78e70](https://github.com/paradigmxyz/reth/commit/9a732f6b7813f9ecb5f28b379d3062c287a78e70)

### CODE REVIEW

File: src/execute.rs [hunk 1]
```diff
@@ -289,9 +325,17 @@ where
     // - Call block rewards contract for bridged xDAI mint
 
     if chain_spec.is_shanghai_active_at_timestamp(block_timestamp) {
-        let withdrawals = withdrawals.ok_or(BlockExecutionError::Other(
-            "block has no withdrawals field".to_owned().into(),
-        ))?;
+        // let withdrawals = withdrawals.ok_or(BlockExecutionError::other::<BlockExecutionError::Internal>::(
+        //     "block has no withdrawals field".to_owned().into(),
+        // ))?;
```
[the remark this answers is not in the document]
Comment (debjit-bw) [reply src/execute.rs:327]: yes, removed

File: src/execute.rs [hunk 2]
```diff
@@ -420,7 +526,7 @@ where
 
     // [Gnosis/fork] Copy paste code from crates/ethereum/evm/src/execute.rs::EthBatchExecutor
     fn finalize(mut self) -> Self::Output {
-        self.stats.log_debug();
+        // self.stats.log_debug();
```
[the remark this answers is not in the document]
Comment (debjit-bw) [reply src/execute.rs:526]: done, removed

File: src/gnosis.rs [hunk 3]
```diff
@@ -54,8 +57,8 @@ where
     let withdrawal_contract_address = chain_spec
         .deposit_contract
         .as_ref()
-        .ok_or(BlockExecutionError::Other(
-            "deposit_contract not set".to_owned().into(),
+        .ok_or(BlockValidationError::DepositRequestDecode(
```
[the remark this answers is not in the document]
Comment (debjit-bw) [reply src/gnosis.rs:60]: ah good catch, i forgot to replace this one with our custom error. fixed ✅

**Review by dapplion — APPROVED** [review]
LGTM, nice work!

**Pull request merged**